quickwit/quickwit-doc-mapper/src/default_doc_mapper/date_time_type.rs

```diff
@@ -206,7 +206,7 @@ mod tests {
                     "rfc3339"
                 ],
                 "output_format": "unix_timestamp_secs",
-                "precision": "milliseconds",
+                "precision": "millisecond",
```

Comment:
I think we should revert this change in tantivy.

Reply:
That's the PR: https://github.com/quickwit-oss/tantivy/pull/2051

`millisecond` or `milliseconds`, it's the same for me (as conclusion I don't think breaking the API is worth it)

`DatePrecision` to `DateTimePrecision` is the correct rename though

Reply:
Let's revert the s part.
Can you review this PR and point to the new main?
https://github.com/quickwit-oss/tantivy/pull/2065